content/locale/en.js

```diff
@@ -357,6 +357,7 @@ const common = {
 	FlaggeyByPARASRemint:
 		'WARNING: This collection has been paused for reminting, please contact the creator',
 	FlaggeyByPARASRugpull: 'This creator has been flagged by Paras due to rug pull',
+	FlaggedByPARASScam: 'This creator has been flagged by Paras due to scam',
```

Comment:
ganti jadi ini gan dari Afiq

"This account is scamming the community. Please be careful when interacting with this account."